programs/survival/bee_healing.sc

```diff
@@ -1,15 +1,24 @@
 // Bees have an annoying habit of drowning themselves, so yeah
 __config()->{'scope' -> 'global'};
 
+global_bees = entity_selector('@e[type=bee]');
+
 begin()->(
-    bees = entity_selector('@e[type=bee]');
-    for (bees, 
-        health = _~'health';
-        if (health < 10,
-            modify(_, 'health', health+1);
+    for (global_bees,
+        if (_ ~ 'removed',
+            delete(global_bees, _i);
```

Comment:
Instead you could also (in the code below) handle the `on_removed` event to remove them.

Reply:
Ok, I've done that